src/Forms/GridField/GridFieldExportButton.php

```diff
@@ -244,7 +244,10 @@ public function generateExportFileData($gridField)
                         $value = $columnHeader($relObj);
                     } elseif ($gridFieldColumnsComponent && in_array($columnSource, $columnsHandled ?? [])) {
                         $value = strip_tags(
-                            $gridFieldColumnsComponent->getColumnContent($gridField, $item, $columnSource) ?? ''
+                            html_entity_decode(
+                                $gridFieldColumnsComponent->getColumnContent($gridField, $item, $columnSource),
+                                ENT_QUOTES
+                            ) ?? ''
```

Comment:
```suggestion
                        $value = Convert::xml2raw(strip_tags(
                            $gridFieldColumnsComponent->getColumnContent($gridField, $item, $columnSource) ?? ''
                        ));
```
This is essentially just undoing the `Convert::raw2xml($value)` step in `GridFieldDataColumns::castValue()` - but even this isn't catching all scenarios.

We probably need to limit this to _only_ undo it if it was done in the first place (i.e. follow the same if/else chain from that method), and to also revert the `nl2br` when appropriate.

It might be cleanest to implement a new method to handle this new chain of if/else statements so it's easier to see what's going on.